.eslintrc.json

```diff
@@ -11,16 +11,13 @@
     "@tinymce/no-enums-in-export-specifier": "off",
     "@tinymce/no-unimported-promise": "off",
     "@typescript-eslint/ban-ts-ignore": "off", // this will be added to the plugin
-    "@typescript-eslint/camelcase": "off",
-    "@typescript-eslint/consistent-type-definitions": "off",
-    "@typescript-eslint/explicit-member-accessibility": "off",
+    "@typescript-eslint/camelcase": "off", // leave off
+    "@typescript-eslint/consistent-type-definitions": "off", // leave off
     "@typescript-eslint/indent": "off",
     "@typescript-eslint/member-delimiter-style": "off",
     "@typescript-eslint/member-ordering": "off",
     "@typescript-eslint/no-empty-function": "off",
-    "@typescript-eslint/no-for-in-array": "off",
-    "@typescript-eslint/no-inferrable-types": "off",
```

Comment:
I wish I had this in reasonml sometimes, beginners (particularly coming from TS) annotate _everything_ unnecessarily :joy: